src/libAtomVM/interop.c

```diff
@@ -201,8 +215,13 @@ InteropFunctionResult interop_iolist_size(term t, size_t *size)
 
     while (!temp_stack_is_empty(&temp_stack)) {
         if (term_is_integer(t)) {
```

Comment:
This branch looks like the same of `term_is_binary(t)` one, so I would just write `if (term_is_integer(t) || term_is_binary(t))`